web-components/src/components/form/CreditSendForm.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { makeStyles } from '@mui/styles';
 import { Box } from '@mui/system';
 import { Field, Form, Formik, FormikErrors } from 'formik';
-import { useTrack } from 'use-analytics';
+import { useTracker } from 'web-registry/src/lib/tracker/useTracker';
```

Comment:
again, it feels like a step in the wrong direction importing web-registry modules into web-components. Prefer to keep all tracking code in the codebase of the app we are tracking. 

Another way we could do this is to simply wrap a web-component in a new web-registry component, and add the tracking logic there. That would mean less repeated code than my previous suggestion.